test/tools/ossfuzz/CMakeLists.txt

```diff
@@ -31,6 +33,16 @@ if (OSSFUZZ)
     target_link_libraries(solc_opt_ossfuzz PRIVATE libsolc evmasm)
     set_target_properties(solc_opt_ossfuzz PROPERTIES LINK_FLAGS ${LIB_FUZZING_ENGINE})
 
+    add_executable(solc_opt_mutator_ossfuzz
+            solc_opt_ossfuzz.cpp
+            ../fuzzer_common.cpp
+            ../../TestCaseReader.cpp
+            SolidityGenerator.cpp
+            SolidityCustomMutatorInterface.cpp
+    )
+    target_link_libraries(solc_opt_mutator_ossfuzz PRIVATE libsolc evmasm)
```

Comment:
Why do we need to link `evmasm` explicitly and why isn't `libsolc` enough?

Reply:
Hmm... Looks like I copied this line from here: https://github.com/ethereum/solidity/blob/aa6e96ce886f87550769dc5cbdec624a49f1fc5b/test/tools/CMakeLists.txt#L8

Without linking in evmasm, fuzzer build complains about not being able to locate `fuzzer_common.h`. Strange.